src/core/software/pupil-capture.md

```diff
@@ -160,56 +164,27 @@ Make sure to always use the **v0.4 marker design** for best detection performanc
-1. Show the marker to the subject and hold the marker still. You will hear a "click" sound when data sampling starts, and one second later a "tick" sound when data sampling stops.
-1. Move the marker to the next location and hold the marker still.
-1. Repeat until you have covered the subject's field of view (generally about 9 points should suffice).
-1. Show the 'stop marker' or press `c` on your keyboard or click the blue circular `C` button in the left hand side of the world window to stop calibration.
-
-You will notice that there are no standard controls, only an `Advanced` sub-menu to control detection parameters of the marker and to debug by showing edges of the detected marker in the world view.
-
-::: tip
-<v-icon large color="info">info_outline</v-icon>
-When should I use the Pupil Calibration <strong>Stop</strong> Marker? - Use the <strong>stop</strong> marker to stop/end a calibration. You can also stop a calibration via the Pupil Capture GUI.</a>
-:::
-
-
-
-#### Single Marker Calibration
-Calibrate using a single marker displayed on screen or hand held marker. Gaze at the center of the marker and move your head in a spiral motion. You can also move your head in other patterns. This calibration method enables you to quickly sample a wide range of gaze angles and cover a large range of your FOV.
-
-1. Select Single Marker Calibration
-1. Press `c` on your keyboard or click the blue circular `C` button on the left hand side of the world window to start calibration.
-1. Look at the center of the marker.
-1. Slowly move your head while gazing at the center of the marker. We have found that a spiral pattern is an efficient way to cover a large area of the FOV.
-1. Press the `C` button on your keyboard  or show the stop marker to stop calibrating.
+1. Select `Single Marker` choreography
+2. Press `c` on your keyboard or click the blue circular `C` button on the left hand side of the world window to start calibration.
+3. Look at the center of the marker.
+4. Slowly move your head while gazing at the center of the marker. We have found that a spiral pattern is an efficient way to cover a large area of the FOV.
+5. Press the `C` button on your keyboard  or show the stop marker to stop calibrating.
 
 ::: tip
 <v-icon large color="info">info_outline</v-icon>
-If you're using a manual marker, make sure to select <code>Marker display mode > manual</code>. Also make sure that you do <strong>not</strong> display the marker on the screen when using a printed marker. Two markers visible to the world view at the same time will result in an inaccurate calibration.
+If you're using a physical marker, make sure to select <code>Physical Marker</code> in the <code>Marker display mode</code>. Also make sure that you do <strong>not</strong> display the marker on the screen at the same time when using a printed marker. Two markers visible to the world view at the same time will result in an inaccurate calibration.
```

Comment:
This is the default now, not sure we need this tip anymore.

Reply:
Agreed. I'll delete this.